src/nxdoc/getting-started/best-practices/choosing-how-to-import-data-in-the-nuxeo-platform.md

```diff
@@ -169,58 +172,71 @@ May it be a file system binary store, an S3 binary store or an Azure Object stor
 
-No out-of-the-box format for metadata values specification. Also not recommended if import rate is the critical factor, since data transit over HTTP/S.
+Files must be available on a file system mounted on the Nuxeo server.
 
-### Nuxeo Platform Importer
+#### Nuxeo Scan Importer
 
 **Features**
 
-The [Nuxeo Bulk Document Importer](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-platform-importer) is an importer framework provided as an addon that can be used to build custom importers. It relies on a standard crawler, transformer, writer schema. The Scan importer and CSV importer addons are using that framework (see next sections). It is the de-facto choice when you want to reach hyperscale numbers with importing content (up to 10&nbsp;000s of documents per second). All you need to do is write your own Document Factory that will be in charge of the document creation logic in the repository. You can then easily launch the import controlling how many documents are done in a batch, how many batches per transaction, etc.
+The [Nuxeo Platform Scan Importer](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-scan-importer) is a submodule of the importer framework and is typically used for the output of a digitalization chain. Nuxeo Platform Scan Importer listens to a given folder and will import all content referenced via XML files, with their metadata, etc. Scan importer also offers very advanced XML <--> documents mapping possibilities, with ability to use some automation processing during the import phase.
 
 **Customization Language**
 
 Java
 
 **Customization Logic**
 
-The importer framework offers many customization possibilities. You can read the [Nuxeo Platform Importer documentation]({{page page='nuxeo-bulk-document-importer'}}) to learn more.
+Scan Importer is configurable via XML extensions for the metadata mapping. The documentation provides links to [simple and advanced use cases]({{page page='scan-documents-importer'}})
 
-**Limitations**
+#### Nuxeo Stream Importer
 
-Files must be available on a file system mounted on the Nuxeo server.
+**Features**
 
-### Nuxeo Scan Importer
+The [Nuxeo Stream Importer](https://github.com/nuxeo/nuxeo/tree/master/addons/nuxeo-platform-importer/nuxeo-importer-stream) defines a producer/consumer pattern and uses the Log features provided by Nuxeo Stream. The Log is used to perform mass import. It decouples the Extraction/Transformation from the Load (using the ETL terminology). The extraction and transformation is done by a document message producer with custom logic. This module comes with a random document and a random blob generator, that does the same job as the random importer of the `nuxeo-importer-core` module. The load into Nuxeo is done with a generic consumer.
+Automation operations are exposed to run producers and consumers.
 
-**Features**
+Two options are offered to perform the import:
+1. Two steps import: Generate and Import documents with blobs.
```

Comment:
Two steps import: Extract the document metadata and blobs then load documents into Nuxeo.
Four steps import: Extract the blobs, load the blobs into Nuxeo, extract the document metadata then load documents into Nuxeo.